adcircpy/mesh/base.py

```diff
@@ -391,9 +298,27 @@ def multipolygon(self) -> MultiPolygon:
 
 class Grd(ABC):
     def __init__(self, nodes, elements=None, description=None, crs=None):
```

Comment:
This essentially assumes that the inputs are pandas dataframes rather than dicts?
I'm not too happy about that assumption, but I will not protest bc I'm not using this package so it won't affect me. But I think it's not a smart move to assume dataframes as inputs.